style.css

```diff
@@ -84,6 +116,204 @@ body {
   list-style: none;
 }
 
+.works-section {
+  background-color: inherit;
+  display: grid;
+  grid-template-columns: 100%;
+  padding: 7.12rem 1.5rem;
+  row-gap: 5.3rem;
+  justify-content: center;
+}
+
+.works-card {
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  padding: 1rem;
+  border-radius: 1rem;
+}
+
+.works-card--img {
+  margin-bottom: 0.75rem;
+}
+
+.works-card--title {
+  font-weight: 700;
+  font-size: 2em;
+  color: #172b4d;
+  margin-bottom: 0.75rem;
+}
+
+.works-card--info {
+  display: flex;
+  justify-content: space-between;
+  padding: 0.5rem 0;
+  margin-right: 3.6rem;
+  margin-bottom: 0.75rem;
+}
+
+.works-card--info li {
+  list-style: none;
+  font-size: 0.8em;
+  font-weight: 600;
+  line-height: 1em;
+  color: #7a869a;
+}
+
+.works-card--info > li.canopy {
+  color: #344563;
+}
+
+.works-card--detail {
+  font-weight: 400;
+  font-size: 0.9em;
+  line-height: 24px;
+  color: #344563;
+  padding-right: 3rem;
+  margin-bottom: 0.75rem;
+}
+
+.works-card--languages {
+  display: flex;
+  justify-content: space-between;
+  margin-right: 5.75rem;
+  margin-bottom: 0.75rem;
+  padding: 0;
+}
+
+.works-card--languages > li {
+  list-style: none;
+  padding: 0.25rem 0.75rem;
+  background-color: #ebebff;
+  border-radius: 0.5rem;
+  color: #6070ff;
+  font-weight: 500;
+  font-size: 0.75em;
+  line-height: 1em;
+  letter-spacing: 0.03em;
+}
+
+.works-card button {
+  align-self: start;
+}
```

Comment:
- [ ] You did a good job styling all the buttons inside the document :+1: , but as a [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) you should add :hover and :active interactions to the design of your project because it will improve the user experience :+1: 
- Consider the image below for reference
![212472166-701382c9-9303-48fb-b0b5-e2a94147dcb6](https://user-images.githubusercontent.com/99394374/212482907-9f7be927-f7ef-4986-9839-a3d01ba68502.png)

Reply:
@AlphaNtihinduka thanks for the replay but I have already made the buttons hoverable and active when clicked.
I think the problem is when we inspect it in a mobile mode but works fine when we inspect it in normal mode(larger screen and on smaller ones). what should I do?

Reply:
Here is my code snippet
![image](https://user-images.githubusercontent.com/51995150/212487084-bcb63127-700e-4d2a-8854-89bfee18c841.png)